src/Tempest/Router/src/functions.php

```diff
@@ -25,4 +25,24 @@ function uri(array|string|MethodReflector $action, mixed ...$params): string
             ...$params,
         );
     }
+
+    /**
+     * Checks whether the given controller action matches the current URI.
+     */
+    function is_uri(array|string|MethodReflector $action, mixed ...$params): bool
```

Comment:
Shouldn't this be called `is_current_uri` then?

Reply:
My thoughts was that `is_uri` as a function makes sense and is clear, while I'd like a more explicit method on the actual router object, hence the longer name. Both have comments anyway.

If the inconsistency is bothering here, I'd rather change `isCurrentUri` on the router to `isUri`, but if it's not an issue, I'd prefer keeping it that way

Reply:
But isn't there a difference? When I read `is_uri`, I expect this function to return true for any string that is a URI. Which isn't the case, it only returns true when it's the _current_ uri>

Reply:
random guy dropping in 🧇 
But could that be named `is_matching_uri`  ? :P or `matches_uri` ? :P

Reply:
> But isn't there a difference? When I read `is_uri`, I expect this function to return true for any string that is a URI. Which isn't the case, it only returns true when it's the _current_ uri?

Very good point actually, I don't know how I missed that. I like `matches_uri` as well but let's keep `is_current_uri` for consistency

Reply:
Thank you random guy @CoolGoose for solving our naming issue :D